lib/components/WizardProgress/__snapshots__/WizardProgress.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`WizardProgress > renders correctly 1`] = `
 <ol
-  class="ds-flex ds-gap-4 ds-text-[24px] ds-text-black ds-font-semibold ds-leading-[24px]"
+  class="ds-flex ds-gap-4 ds-text-black sm:ds-text-heading-2 sm:ds-leading-[24px] ds-text-heading-3-mobile ds-leading-[16px]"
```

Comment:
Are these pixel values necessary?

Reply:
When tested, it is true that in current situation it does not cause harm when the line-height is not explicitly told and using the same as given in the design.

This was done for the code/implementation to match with the current state of design and it might be annoying to notice later that this does not match with the design when checking later.
And then putting comment to the code that this can be ignored is more of a hassle than just leave these values.

**BUT** I can check if there would be some nice solution to have these leading-values as tokens also and use them. It seems that they vary a lot in the current tokens already and in the code.